Plans/visionOS-Swift_Base.xctestplan

```diff
@@ -0,0 +1,18 @@
+{
+  "configurations" : [
```

Comment:
`m`: Do we really need a testplan for visionOS and watchOS. We could ditch these if possible, because we don't run tests on these platforms.